pkg/config/module.go

```diff
@@ -13,3 +17,15 @@ func PackageVersionedName(module, version, ext string) string {
 func FmtModVer(mod, ver string) string {
 	return fmt.Sprintf("%s@%s", mod, ver)
 }
+
+// ModuleVersionFromPath returns module and version from a
+// storage path
+// E.g athens/@v/v1.0.info -> athens and v.1.0
+func ModuleVersionFromPath(path string) (string, string) {
+	segments := strings.Split(path, "/@v/")
```

Comment:
can this be a const?

Reply:
Given the current state I am not sure if it would be more readable or not. The counterpart of this function (PackageVersionedName) is in the same file. If we decide to put make it a constant I would change that as well. Not sure it would be better.